rskj-core/src/main/java/co/rsk/peg/whitelist/constants/WhitelistConstants.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.peg.whitelist.constants;
+
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+
+public class WhitelistConstants {
```

Comment:
I see that the intention here is to have this class be extended by a mainnet, testnet, etc. Would it make sense to have this class as an abstract class?

Reply:
@apancorb, Amazing comment! Indeed, it shouldn't be an abstract class. It should be an Interface. Some weeks ago, I was discussing this topic with @marcos-iov  since in these cases we should apply a behavioral design pattern, Strategy! It would be interesting to discuss this at a technical meeting. WDYT, @marcos-iov ?